ddtrace/contrib/internal/botocore/patch.py

```diff
@@ -104,6 +104,17 @@ def _load_dynamodb_primary_key_names_for_tables() -> Dict[str, Set[str]]:
         "empty_poll_enabled": asbool(os.getenv("DD_BOTOCORE_EMPTY_POLL_ENABLED", default=True)),
         "dynamodb_primary_key_names_for_tables": _load_dynamodb_primary_key_names_for_tables(),
         "add_span_pointers": asbool(os.getenv("DD_BOTOCORE_ADD_SPAN_POINTERS", default=True)),
+        "payload_tagging_request": os.getenv("DD_TRACE_CLOUD_REQUEST_PAYLOAD_TAGGING", default=None),
+        "payload_tagging_response": os.getenv("DD_TRACE_CLOUD_RESPONSE_PAYLOAD_TAGGING", default=None),
+        "payload_tagging_max_depth": int(
+            os.getenv("DD_TRACE_CLOUD_PAYLOAD_TAGGING_MAX_DEPTH", 10)
+        ),  # RFC defined 10 levels (1.2.3.4...10) as max tagging depth
+        "payload_tagging_max_tags": int(
+            os.getenv("DD_TRACE_CLOUD_PAYLOAD_TAGGING_MAX_TAGS", 758)
+        ),  # RFC defined default limit - spans are limited past 1000
+        "payload_tagging_services": set(
+            os.getenv("DD_TRACE_CLOUD_PAYLOAD_TAGGING_SERVICES", default={"s3", "sns", "sqs", "kinesis", "eventbridge"})
+        ),
```

Comment:
Could we also update our [configuration docs](https://github.com/DataDog/dd-trace-py/blob/main/docs/configuration.rst) with these new configs (name, brief description, default value(s), which version it was introduced in, etc.)